client/src/app/site/pages/meetings/pages/participants/pages/participant-detail/components/participant-detail-view/participant-detail-view.component.ts

```diff
@@ -323,7 +323,24 @@ export class ParticipantDetailViewComponent extends BaseMeetingComponent {
                           .filter(id => !!id)
                     : []
             };
-            await this.repo.update(payload, this.user!).concat(this.repo.setPresent(isPresent, this.user!)).resolve();
+            const title = this.translate.instant(`This action will remove you from one or more groups.`);
+            const content = `This may diminish your ability to do things in this meeting and you may not be able to revert it by youself. Are you sure you want to do this?`;
```

Comment:
This should be translatable.
```suggestion
            const content = _(`This may diminish your ability to do things in this meeting and you may not be able to revert it by youself. Are you sure you want to do this?`);
```